src/components/DateTimeRange.js

```diff
@@ -31,7 +31,7 @@ const DateTimeRange = () => {
 	const [dateTimeEnd, setDateTimeEnd] = useState();
 	const [timezone, setTimezone] = useState();
 
-	subscribe(saveDateTime);
+	saveDateTime();
```

Comment:
The subscribe calls the endpoint a LOT of times. Firing it just once here fires only once. This will all be looked at when the Event Date block is worked on. I'm planning on doing that next.